src/CONST.ts

```diff
@@ -1343,6 +1344,24 @@ const CONST = {
         },
     },
 
+    SAGE_INTACCT_CONFIG: {
+        MAPPING_VALUE: {
+            NONE: 'NONE',
+            DEFAULT: 'DEFAULT',
+            TAG: 'TAG',
+            REPORT_FIELD: 'REPORT_FIELD',
+        },
```

Comment:
These are not keys of the config, so I think it would be better to pull these out into their own constants, similar to what we've done with NetSuite. If so, maybe `SAGE_INTACCT_MAPPING_VALUE` would be the object name?